packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -30,125 +24,23 @@ import {
 } from '../../../src/commands/commandConstants';
 import * as commandUtils from '../../../src/commands/commandUtils';
 import {
-  DeviceQuickPickItem,
-  directoryLevelUp,
-  forceLightningLwcPreview,
-  getProjectRootDirectory,
-  PlatformName,
-  platformOptions
+  forceLightningLwcPreview
 } from '../../../src/commands/forceLightningLwcPreview';
+import {
+  desktopPlatform,
+  androidPlatform,
+  iOSPlatform,
+  OperationCancelledException,
+  LWCPlatformQuickPickItem,
+  LWCUtils
+} from '../../../src/commands/lwcUtils';
 import { nls } from '../../../src/messages';
 import { DevServerService } from '../../../src/service/devServerService';
 import { WorkspaceUtils } from '../../../src/util/workspaceUtils';
-import {
-  ChannelService,
-  notificationService
-} from '@salesforce/salesforcedx-utils-vscode';
-
-const sfdxDeviceListCommand = 'force:lightning:local:device:list';
```

Comment:
Again, many of the deleted codes in this file are actually refactored/moved to the new `lwcUtils.test.ts`